packages/errors/src/index.ts

```diff
@@ -41,15 +41,11 @@ export interface ErrorWithOptionalCode extends Error {
   code?: string;
 }
 
-export interface ErrorWithName extends Error {
-  name: string;
-}
-
 /**
  * Test to see if a given exception is an AWS Throttling Exception
  */
 export const isThrottlingException = (err: ErrorWithOptionalCode) =>
-  err.code === 'ThrottlingException';
```

Comment:
I think we need tests for this. It's concerning that the V3 update broke it and we didn't have anything to notify us.

Reply:
Does the mocking package we're using help us here? Maybe it will return an error with the correct V3 format we can test against? https://www.npmjs.com/package/aws-sdk-client-mock

I agree that we don't want to actually cause a 'real' throttling exception. Even if we have to write our own error object, it feels like there could be value there in saying basically "this is what we think the errors look like at the time of writing".

Reply:
Also, this is a long shot but does the SDK provide a version of these functions? Like can we import something similar `isThrottlingException` that they manage?

Reply:
If there's not an easy way to do this, perhaps we punt testing to the additional ticket you created for following up on this more generally. I do get that it's a production issue so should be done quickly.